examples/cms-contentstack/README.md

```diff
@@ -0,0 +1,131 @@
+# Create a Static Blog Site Using Next.js and Contentstack
+
+This example demonstrates how you can set up and run a server side example using Next.js and Contentstack with minimal steps. [SSR](https://nextjs.org/docs/basic-features/pages) and [Contentstack-CMS](https://www.contentstack.com/) as the data source.
+
+## Demo
+
+[https://cms-contentstack-starter](https://cms-contentstack-starter.vercel.app/)
```

Comment:
[https://cms-contentstack-starter.vercel.app](https://cms-contentstack-starter.vercel.app/) (it's missing the "vercel.app" suffix)